scripts/automation/dor-dod-templates.js

```
@@ -0,0 +1,335 @@
+/**
+ * Detect issue type from labels array
+ * @param {array} labels - Labels array from GitHub API
+ * @returns {string|null} Type label (e.g., 'type:bug') or null if none found
+ */
+function detectTypeFromLabels(labels) {
+  if (!Array.isArray(labels)) return null;
+  const typeLabel = labels.find(l => l.name && l.name.startsWith('type:'));
+  return typeLabel ? typeLabel.name : null;
+}
+
+module.exports = {
```

ESLint

[error] 'module' is not defined.

(no-undef)